src/test/osd/osd-fast-mark-down.sh

```diff
@@ -62,28 +63,43 @@ function test_fast_kill() {
    killid=0
    previd=0
 
-   # kill random osd and see if 1 sec after, the osd count decreased.
+   # kill random osd and see if after max MAX_PROPAGATION_TIME, the osd count decreased.
    for i in {1..2}; do
      while [ $killid -eq $previd ]; do
         killid=${pids[$RANDOM%${#pids[@]}]}
      done
      previd=$killid
 
      kill -9 $killid
-     sleep 1
+     time_left=$MAX_PROPAGATION_TIME
+     
+     while [ $time_left -gt 0 ]; do
```

Comment:
what if it times out? we might want to init the `down_osds ` with 0 before the `while` loop.